lib/github_changelog_generator/parser.rb

```diff
@@ -205,6 +208,7 @@ def self.default_options
         enhancement_labels: %w(enhancement Enhancement),
         bug_labels: %w(bug Bug),
         exclude_labels: %w(duplicate question invalid wontfix Duplicate Question Invalid Wontfix),
+        issue_line_labels: [],
```

Comment:
The `KNOWN_OPTIONS` constant needs this setting name as well:

https://github.com/skywinder/github-changelog-generator/blob/master/lib/github_changelog_generator/options.rb#L7